src/entity.js

```diff
@@ -175,7 +176,7 @@ export class Entity {
 
     return this.getTransport()
       .update(this.getResource(), this.getId(), requestBody)
-      .then((updated) => response = updated)
+      .then(updated => { response = updated; })
```

Comment:
Mhm. I kind of get it, but it's convenient here. So by rule, it should then be an assignment followed by a return?

Either way, I think we can lose the curly braces, and the semicolon?